src/utility/svd.jl

```diff
@@ -34,20 +36,95 @@ SVD from `KrylovKit.svdsolve` is used.
 """
 PEPSKit.tsvd(t, alg; kwargs...) = PEPSKit.tsvd!(copy(t), alg; kwargs...)
 function PEPSKit.tsvd!(t, alg::SVDAdjoint; trunc::TruncationScheme=notrunc(), p::Real=2)
-    return TensorKit.tsvd!(t; alg=alg.fwd_alg, trunc, p)
+    return TensorKit.tsvd!(t; alg, trunc, p)
+end
+
+# TODO: add `LinearAlgebra.cond` to TensorKit
+# Compute condition number smax / smin for diagonal singular value TensorMap
+function _condition_number(S::AbstractTensorMap)
+    smax = maximum(first ∘ last, blocks(S))
+    smin = maximum(last ∘ last, blocks(S))
+    return smax / smin
+end
+
+# Copy code from TensorKit but additionally return full U, S and V to make compatible with :fixed mode
+function TensorKit._tsvd!(
+    t::TensorMap{<:RealOrComplexFloat}, alg::SVDAdjoint, trunc::TruncationScheme, p::Real=2
+)
+    U, Σ, V⁺, truncerr = tsvd(t; trunc=NoTruncation(), p, alg)
```

Comment:
```suggestion
    U, Σ, V⁺, truncerr = PEPSKit.tsvd(t; trunc=NoTruncation(), p, alg)
```
I think this as causing the stack overflow in the tests?